README.md

```diff
@@ -112,7 +129,7 @@ Each PR, especially those which increase the line count, should have a descripti
   * See [lazy.nvim uninstall](https://github.com/folke/lazy.nvim#-uninstalling) information
 * Are there any cool videos about this plugin?
   * Current iteration of kickstart (coming soon)
-  * Here is one about the previous iteration of kickstart: [video introduction to Kickstart.nvim](https://youtu.be/stqUbv-5u2s).
+  * Here is one about the previous iteration of kickstart: [video introduction to Kickstart.nvim](https://youtu.be/stqUbv-5u2s). Note the install via init.lua no longer works.
```

Comment:
I think signposting this video as obsolete is a really good call, but might it be better to just say something like "Note the install instructions here no longer work. Just follow the instructions above instead" or similar?

Reply:
The rest looks good. I'm really concerned that this particular turn of phrase will confuse newcomers.  Please consider changing this to just ask folks to follow these instructions instead and I'll merge this in a jiffy :)

Reply:
OK, this is excellent work. I think I'm going to merge and then make that one last wording tweak around the out of date install information in the video myself. Thanks a bunch!